packages/botonic-core/src/core-bot.ts

```diff
@@ -90,6 +90,15 @@ export class CoreBot {
     lastRoutePath,
   }: BotRequest): Promise<BotResponse> {
     const output = await this.runInput({ input, session, lastRoutePath })
+
+    if (session._botonic_action?.startsWith('flow_builder_bot_action:')) {
+      return await this.runFollowUpFlowBuilderBotAction(output, {
```

Comment:
Let's change these names to something generic and not related to "Flow Builder". The core package should not know that the flow builder exists, that's why we have a separate package for that. Otherwise is like introducing a circular dependency. I propose to call it `redirect_action`, the same with the function name.